packages/admin-ui/src/components/Filter/filter.tsx

```diff
@@ -0,0 +1,83 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import { Button } from '../Button'
+
+import { VisuallyHidden } from '../VisuallyHidden'
+import type {
+  FilterItem,
+  UseMultipleFilterReturn,
+} from './useMultipleFilterState'
+import { PopoverDisclosure } from 'reakit/Popover'
+import { PopoverFooter, Popover } from './popover'
+import { tag } from '@vtex/admin-ui-react'
+import { IconCaretUp } from '@vtex/phosphor-icons'
+import { MultipleSelectContent } from './multiple-select-content'
+
+export function Filter(props: FilterProps) {
+  const { state } = props
+  const { onClear, onChange, popover, selectedValues, label, labelProps } =
+    state
+
+  const selectedItemsLabel =
+    selectedValues.length &&
+    `${selectedValues[0]}${
+      selectedValues.length > 1 ? `, +${selectedValues.length - 1}` : ''
+    }`
```

Comment:
suggestion: I think like this could it be easier to understand the code from above

```suggestion
  const selectedItemsLabel = selectedValues.length > 1
     ? `${selectedValues[0]}, +${selectedValues.length - 1}`
     : selectedValues[0]
```

Reply:
🤔 i don't know if this really helps... because when selectedValues.length === 0 nothing on the left matters...